src/sqlancer/databend/DatabendErrors.java

```diff
@@ -0,0 +1,116 @@
+package sqlancer.databend;
+
+import sqlancer.common.query.ExpectedErrors;
+
+public final class DatabendErrors {
+
+    private DatabendErrors() {
+    }
+
+    public static void addExpressionErrors(ExpectedErrors errors) {
+        errors.add("with non-constant precision is not supported");
+        errors.add("Like pattern must not end with escape character");
+        errors.add("Could not convert string");
+        errors.add("ORDER term out of range - should be between ");
+        errors.add("You might need to add explicit type casts.");
+        errors.add("can't be cast because the value is out of range for the destination type");
+        errors.add("field value out of range");
+        errors.add("Unimplemented type for cast");
+
+        errors.add("Escape string must be empty or one character.");
+        errors.add("Type mismatch when combining rows"); // BETWEEN
+
+        errors.add("invalid UTF-8"); // TODO
+        errors.add("String value is not valid UTF8");
+
+        errors.add("Invalid TypeId "); // TODO
+
+        errors.add("GROUP BY clause cannot contain aggregates!"); // investigate
+
+        addRegexErrors(errors);
+
+        addFunctionErrors(errors);
+
+        errors.add("Overflow in multiplication");
+        errors.add("Out of Range");
+        errors.add("Date out of range");
+
+        // collate
+        errors.add("Cannot combine types with different collation!");
+        errors.add("collations are only supported for type varchar");
+
+        // // https://github.com/cwida/Databend/issues/532
```

Comment:
There is a more elegant pattern for this, see https://github.com/sqlancer/sqlancer/blob/master/src/sqlancer/tidb/TiDBBugs.java. You don't need to adopt this for this PR, but this might be a potential follow-up improvement. The idea is to make it explicit which code depends on a fix to an issue, so later on, it might be easier to remove this code/error once the issue has been addressed.